### PR TITLE
Feat/basics

### PR DESCRIPTION
**Pull request by sievertknorrblad**

### CODE REVIEW

**Review by iamstarkov — CHANGES_REQUESTED** [review]
can you add `"repository": "https://github.com/user/repo/",`, where user is your GitHub user and repo is your repo name. so when you run `nom repo` in a terminal it will open a GitHub repo for you

File: src/components/footer/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    { text: 'Pure UI',          url: 'https://rauchg.com/2015/pure-ui' },
+    { text: 'Documentation',    url: 'https://reactjs.org/docs/' },
+    { text: 'PropTypes',        url: 'https://reactjs.org/docs/typechecking-with-proptypes.html' },
+    { text: 'Create react app', url: 'https://github.com/facebook/create-react-app/' },
+    { text: 'github',           url: 'https://github.com/user/repo' },
+  ];
+
+export const Footer = () => (
+  <div className="footer">
+    {additionalMenu.map(o => (<a key= {o.text} href={o.url}>{o.text}  </a>))}
```
Comment (iamstarkov) [inline src/components/footer/index.js]: what does `o` stand for?

Comment (sievertknorrblad) [reply src/components/footer/index.js]: object i thought. How would you name it? 'linkItem'?

Comment (iamstarkov) [reply src/components/footer/index.js]: `item`, or `x` if you like FP

**Review by iamstarkov — APPROVED** [review]
looks good to me! 👍

**Pull request merged**